src/cpp/session/SessionModuleContext.cpp

```diff
@@ -928,6 +928,11 @@ FilePath oldScopedScratchPath()
    else
       return userScratchPath();
 }
+
+Error rLibsUser(std::string* pRLibsUser)
+{
+   return r::exec::RFunction("Sys.getenv").addParam("x", "R_LIBS_USER").call(pRLibsUser);
```

Comment:
We're in process with R (we use it as a library) so we share an environment block. If that doesn't align with your observations let me know and let's take a closer look!